apps/src/turtle/skins.js

```diff
@@ -39,38 +39,50 @@ exports.load = function (assetUrl, id) {
     }
   };
 
+  /**
+   * Generates a mapping of sticker names to the urls of their images.
+   *
+   * @return the mapping of names to urls
+   */
+  var stickers = function() {
+
+    // Playlab characters
+    var playlab = ['Alien'   , 'Bat'     , 'Bird'     , 'Cat'       , 'Caveboy'   ,
+                   'Cavegirl', 'Dinosaur', 'Dog'      , 'Dragon'    , 'Ghost'     ,
+                   'Knight'  , 'Monster' , 'Ninja'    , 'Octopus'   , 'Penguin'   ,
+                   'Pirate'  , 'Princess', 'Robot'    , 'Soccerboy' , 'Soccergirl',
+                   'Spacebot', 'Squirrel', 'Tennisboy', 'Tennisgirl', 'Unicorn'   ,
+                   'Witch'   , 'Wizard'  , 'Zombie'   ];
+
+    // Miscellaneous stickers
+    var misc = ['Beaver', 'Bunny'      , 'Chicken', 'Elephant', 'Giraffe',
+                'Goat'  , 'Grasshopper', 'Hippo'  , 'Lion'    , 'Llama'  ,
+                'Monkey', 'Moose'      , 'Mouse'  , 'Owl'     , 'Peacock',
+                'Rocket', 'Triceratops', 'Turtle' , 'Zebra'   ];
+
+    var mapping = {};
+
+    var i, name;
```

Comment:
These are defined outside of the loop to make it clear that they are intended to be used in multiple "scopes" (in this loop and the next)